recipes/controller.rb

```diff
@@ -16,14 +16,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+vnc_bind_int = node['osl-openstack']['vnc_bind_interface']['controller']
+node.default['openstack']['endpoints']['compute-vnc-bind']['bind_interface'] =
```

Comment:
While we're at it, please make this `= Chef::DelayedEvaluator.new { node['osl-openstack']['vnc_bind_interface']['controller'] }`